.env

```diff
@@ -0,0 +1,2 @@
+NOTION_SECRET="secret_3nG9kNTECmX3gA7RFCoHCgYR8XAnQzAHv9rItdKXx1I"
+NOTION_DATABASE_ID="0aac587c9c8a4ad9b52c7a138efb3111"
```

Comment:
The .env shall not be commited. 

You can add a .env.example with empty data

This is a huge security risk (The problem now is in the history of github lol)